### ISSUE TITLE
Reconstruction loss

### ISSUE THREAD

**Issue by Animatory** [odegeasslbc/FastGAN-pytorch#21]
Thanks for your work!

In the paper, you stated that reconstruction loss is just MSE, but later in the paper and in the code, you use LPIPS. So was the model trained with LPIPS instead of MSE?

**Comment by ryusukecp3** [comment]
I'm also interested in it.

**Comment by odegeasslbc** [comment]
Hi, in the code I used both mse and lpips. I find that at the beginning of the training, using lpips in addition to mse makes the model converges faster. However, in a long run, it does not affect the final model performance much. Moreover, on different datasets, the model shows different behavior when trained with different settings (mse only, lpips only, or combine both), but the final model performance does not vary much according to this setting. So I think just setting it to be trained by both lpips and mse is the safe way to go.
